vars/sectionCI.groovy

```diff
@@ -35,35 +35,42 @@ def call(params) {
     withAksClient(subscription, environment) {
       def acr = new Acr(this, subscription, env.REGISTRY_NAME, env.REGISTRY_RESOURCE_GROUP)
       def dockerImage = new DockerImage(product, component, acr, new ProjectBranch(env.BRANCH_NAME).imageTag(), env.GIT_COMMIT)
-
       onPR {
         acr.retagForStage(DockerImage.DeploymentStage.PR, dockerImage)
+      }
+      if (config.deployToAKS) {
 
-        if (config.deployToAKS) {
-          withTeamSecrets(config, environment) {
-            stage('Deploy to AKS') {
-              pcr.callAround('aksdeploy') {
-                timeoutWithMsg(time: 15, unit: 'MINUTES', action: 'Deploy to AKS') {
+        withTeamSecrets(config, environment) {
+          stage('Deploy to AKS') {
+            pcr.callAround('aksdeploy') {
+              timeoutWithMsg(time: 15, unit: 'MINUTES', action: 'Deploy to AKS') {
+                onPR {
                   deploymentNumber = githubCreateDeployment()
+                }
 
-                  aksUrl = aksDeploy(dockerImage, params)
-                  log.info("deployed component URL: ${aksUrl}")
+                aksUrl = aksDeploy(dockerImage, params)
```

Comment:
we need to think about binning this soon

Reply:
yeah, only few seem to be left in migration :
https://tools.hmcts.net/confluence/display/CNP/Helm+Charts+Migration+Progress